apollo-api/src/main/java/com/spotify/apollo/RequestContext.java

```diff
@@ -52,23 +52,7 @@
   Map<String, String> pathArgs();
 
   /**
-   * Get the arrival time of the incoming request in nanoseconds. Note that this is not
-   * unix epoch as the time is provided by {@link System#nanoTime()}. To get unix epoch
-   * time, do something like:
-   *
-   * <pre>
-   * {@code
-   * long processingTimeNanos = System.nanoTime() - requestContext.arrivalTimeNanos();
-   * long arrivalTimeUnixEpochMillis = System.currentTimeMillis() +
-   *                                   TimeUnit.NANOSECONDS.toMillis(processingTimeNanos);
-   * }
-   * </pre>
-   *
-   * @see System#nanoTime()
+   * Returns the metadata available for this request.
    */
-  default long arrivalTimeNanos() {
```

Comment:
I would prefer to simplify/remove this rather than deprecate because:

- it's a part of a major version upgrade, 
- it's an easy change to make, and
- it's not normally used by application code, but only by framework-type code (like our Hermes implementation)